package.json

```diff
@@ -41,6 +41,7 @@
     "npm": "^9"
   },
   "jest": {
+    "showSeed": true,
```

Comment:
Interesting, do we currently have flaky tests? Would be good to add a card to make those more stable.

Reply:
I haven't spotted any, but my thought was that it's better to have the seed when you spot one, than not have it and not be able to reproduce it (until it randomly pops up again)